source/projects/jungle_beat/iteration_1.md

```diff
@@ -1,27 +1,24 @@
-# Iteration 1 - Append, All and Count (Single Node)
+# Iteration 1 - Append, All and Count
 
-Great! We have nodes. Next step is to build out the `JungleBeat` class which will manage the list.
+Great! We have nodes. Next step is to build out the `LinkedList` class.
 
-We are also going to build the `append`, `all` and `count` methods. `append` will add another node to the end of the list, `all` will return the data of the linked list in order, and `count` will return the number of nodes in the list.
+We are also going to build the `all` and `count` methods. `all` will return the data of the linked list in order, and `count` will return the number of nodes in the list.
+
+Let's also make sure that we can append multiple nodes using the `append` method.
 
 Expected behavior:
 
 ```ruby
-> require "./lib/jungle_beat"
-> jb = JungleBeat.new("deep")
-=> 1 # number of nodes inserted
-> jb.head
-=> <Node data="deep" next_node=nil #4567890456789456789>
-> jb.head.data
-=> "deep"
-> jb.count
-=> 1
-> jb.append("plop")
-=> 1
-> jb.count
+> require "./lib/linked_list"
+> LL = LinkedList.new
+> LL.append("plop pa")
```

Comment:
let's let the `LinkedList` make no assumptions about the structure of data being passed in -- if you give it "dee dee" that should give you 1 node with "dee dee" as its data

also i dont know why i was using `LL` to demonstrate when we were on slack; just call it `list` or something